src/Paket.Core/Installation/UpdateProcess.fs

```diff
@@ -10,6 +10,8 @@ open Chessie.ErrorHandling
 open Paket.Logging
 open InstallProcess
 
+type RTC = DependencyChangeDetection.ResolutionTriggeringChange
```

Comment:
This is only to bring in the type to avoid having to fully qualify 

The alternatives I currently see are
- opening the DependencyChangeDetection module; not sure if we'd want to do that here
- specifying the type and module every time in the pattern match, which seems ugly
- moving the original type somewhere else, like to `Paket.Domain`; not sure if that would be correct